toolkit/tools/specreader/specreader.go

```diff
@@ -606,3 +626,12 @@ func filterOutDynamicDependencies(pkgVers []*pkgjson.PackageVer) (filteredPkgVer
 
 	return
 }
+
+// convertToToolchainRpmPath updates the .rpm path to point the the toolchain directory rather than the normal out/RPMs
+// directory
+func convertToToolchainRpmPath(currentRpmPath, arch string, toolchainDir string) (toolchainPath string) {
+	dir := filepath.Join(toolchainDir, arch)
+	toolchainPath = filepath.Join(dir, filepath.Base(currentRpmPath))
+	logger.Log.Debugf("Toolchain changing %s to %s", currentRpmPath, toolchainPath)
```

Comment:
Let's stick to surrounding variables one way or another:

```suggestion
	logger.Log.Debugf("Toolchain changing '%s' to '%s'.", currentRpmPath, toolchainPath)
```

or:
```suggestion
	logger.Log.Debugf("Toolchain changing (%s) to (%s).", currentRpmPath, toolchainPath)
```

Reply:
done